mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -200,12 +200,15 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_1506 handleGlErrorByDisableVsync (IJ)V
 		ARG 1 error
 		ARG 2 description
-	METHOD method_1507 openScreen (Lnet/minecraft/class_437;)V
-		COMMENT Opens a new screen, changing the current screen if needed.
+	METHOD method_1507 setScreen (Lnet/minecraft/class_437;)V
+		COMMENT Sets the current screen to a new screen.
 		COMMENT
 		COMMENT <p>If the screen being opened is {@code null} and the client is not in game, the title screen will be opened.
-		COMMENT If the currently opened screen is {@code null} and player is dead then the death screen will be opened.
-		COMMENT Otherwise the currently open screen will be closed.
+		COMMENT If the currently opened screen is {@code null} and player is dead, then the death screen will be opened.
+		COMMENT Otherwise, the currently open screen will be closed.
```

Comment:
I suggest replace `currently` with `previous` to make it more clear (not confuse current with the passed parameter)